spec.md

```diff
@@ -2339,6 +2350,11 @@ Otherwise `NodeExpandVolume` MUST be called after successful `NodePublishVolume`
 
 If a plugin only supports expansion via the `VolumeExpansion.OFFLINE` capability, then the volume MUST first be taken offline and expanded via `ControllerExpandVolume` (see `ControllerExpandVolume` for more details), and then node-staged or node-published before it can be expanded on the node via `NodeExpandVolume`.
 
+The `staging_target_path` field is not required, for backwards compatibility, but the CO SHOULD supply it
+when possible. Plugins can use this field to determine if `volume_path` is where the volume is
+published or staged, and setting this field to non-empty allows plugins to function with less
+stored state on the node.
```

Comment:
We should make this more explicit -- if  `staging_target_path` is specified, then `volume_path` MUST contain the publish path NOT stage path.